browser/src/slideshow/SlideShowPresenter.ts

```diff
@@ -603,7 +621,7 @@ class SlideShowPresenter {
 	}
 
 	/// returns true on success
-	_onPrepareScreen(inWindow: boolean) {
+	_onPrepareScreen(inWindow: boolean, options: any = undefined) {
```

Comment:
option parameters doesn't seem to be used

Reply:
it is still being used
TS2554: Expected 1 arguments, but got 2

Reply:
Yep it's `_onStartInWindow` which calls it with 2 arguments; what is not clear is the goal, given that the parameter is not used in `_onPrepareScreen` maybe do you plan to use it later ?

Reply:
So, I see that you pass the `noClick`, `noKey` options to `SlideShowPresenter._onStartInWindow` in `PresenterConsole._onPresentInConsole` by firing `newpresentinwindow`. Anyway they are not used: these options never reach `_createPresenterHTML`, `_createCanvas` and `_doInWindowPresentation`. 
In fact the slideshow accepts direct input. Moreover current and next slides in the presenter console are updated properly. In the end I don't see the need to prevent input on the window/canvas where the slideshow is running as far as the presenter console is kept in sync.

Reply:
Ok, I thought the presenter console would have full control. I will remove the options.

Reply:
Thanks for fixing that!